epifor/gleam/simulation.py

```diff
@@ -59,6 +59,10 @@ def load_sim(self, path):
             s.definition.get_seasonality(),
             s.definition.get_traffic_occupancy(),
         )
+        s.definition.param_beta = k[0]
```

Comment:
Yeah, this is probably really something for Tomas to answer.

Reply:
Isn't too nice . but working :)

Reply:
I would prefer NOT to have this here in attributes:
* Duplicating the data - which os the principal store? You need to update it in all the setters.
* People may set it and expect change in XML
* The speedup by not querying the (already loaded) XML tree is negligible.